### DISCUSSION TITLE
Stop-publication-esentool NOTICE_NOT_FOUND

### DISCUSSION THREAD

**Discussion by AlexAS81** [OP-TED/eForms-SDK#691] [Q&A] [answered]
We have made the following flow in preview:

a) Send a notice and it has remained in SUBMITTED status.
b) We have stopped the notice using the stop-publication-esentool operation (sending the identifier 3733dfc9-68da-480b-897d-9e657afc090b-01).
c) We have submitted again the same notice with the same noticeId and new version 02 (3733dfc9-68da-480b-897d-9e657afc090b-02).
d) We hace tried to stop the 3733dfc9-68da-480b-897d-9e657afc090b-02 notice and we get the following message:

<xml-fragment>{"code":"notice.NOTICE_NOT_FOUND","message":"Notice not found in TEDMON22","showDetails":false}</xml-fragment>

When a notice is stopped, can the noticeId not be used again?

Thanks in advance,
Alex

**Comment by ferrakaop** [comment, marked as answer]
Please send this issue to ted@publications.europa.eu to check the situation in the Preview environment. This isn't related to the SDK.
